.circleci/config.yml

```
@@ -80,6 +80,24 @@ jobs:
           paths:
             - ~/.m2
 
+  test-bb:
+    executor: clojure
+    steps:
+      - run:
+          name: Install babashka
+          command: |
+            sudo bash < <(curl -s https://raw.githubusercontent.com/babashka/babashka/master/install)
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-test-cljs-{{ checksum "project.clj" }}
+            - v1-test-cljs-
```

This should use a babashka-specific prefix, but could fall back to `v1-test-` as a generic prefix I think, since it'd pull alphabase in all profiles.
```suggestion
            - v1-test-bb-{{ checksum "project.clj" }}
            - v1-test-bb-
```